### ISSUE TITLE
having mixed monitor layout (landscape and portrait) break set-wallpaper contract or simply can't set wall paper any more

### ISSUE THREAD

**Issue by stucash** [elementary/portals#99]
I am running Juno and I have a 3-monitor-setup with two in portrait and one in landscape. 

![screenshot from 2018-11-24 16-42-00](https://user-images.githubusercontent.com/6358866/48970785-65048600-f008-11e8-80c1-d4a7428f9e2f.png)

Display setting shows this layout:

![selection_001](https://user-images.githubusercontent.com/6358866/48970802-95e4bb00-f008-11e8-826c-b123b29bef36.png)

before I change my layout using `xrandr` commands, I can change wallpaper by right clicking on it and do set to background, or just use app like Fondo or Variety. 

After display setting takes effect, nothing above works any more. 

Loki failed on this one when it was just released but fixed it after many system updates. 

appreciate any help.

basically `io.elementary.contract.set-wallpaper` failed. I ran it on command line didn't work either. 

what I did was:

`/usr/lib/x86_64-linux-gnu/io.elementary.contract.set-wallpaper '///usr/share/backgrounds/Ashim DSilva.jpg'`

system log did not issue any error message and on terminal there was no feedback at all.

**Comment by danirabbit** [comment]
Moving to portals since we have a portal and don't use the contact anymore. This might ultimately be a Gala issue though
